.travis.yml

```diff
@@ -27,8 +27,8 @@ matrix:
       install:
        - C:\\Python35\\python -m pip install -r requirements-dev.txt
       script:
-       - C:\\Python35\\python ./bin/run_tests.py
+       - C:\\Python35\\python -m pytest -vv
 
 install: $PYTHON -m pip install -r requirements-dev.txt
 
-script: $PYTHON ./bin/run_tests.py
+script: $PYTHON -m pytest -vv
```

Comment:
Can/should we move `-vv` to somewhere inside `conftest.py`, if these are our prefered defaults?

Reply:
(The cool thing about being able to call `pytest` directly it's simpler, so it would be great if we can forget about these flags :-) )

Reply:
again, easy to do, in pytest.ini

Reply:
Oh, nice. I haven't convinced myself yet that it would be the right thing to do, though

Reply:
done